BUILD.bazel

```diff
@@ -1,15 +1,22 @@
-load("@rules_cuda//cuda:defs.bzl", "cuda_library")
+load("@rules_cc//cc:defs.bzl", "cc_binary", "cc_library")
 
 package(default_visibility = ["//visibility:public"])
 
-cuda_library(
-    name = "kernel",
-    srcs = ["kernel.cu"],
-    hdrs = ["kernel.h"],
+cc_library(
+    name = "hip_info",
+    srcs = ["hip_info.cpp"],
+    hdrs = ["hip_info.hpp"],
+    copts = [
+        # allow warnings from external code
```

Comment:
Can we move this to hip_cpu.BUILD.bazel? I think it will get inherited by everything that depends on it.

Reply:
The setting here in `copts` doesn't propagate:
https://bazel.build/reference/be/c-cpp#cc_binary.copts

Also it looks like `hip_cpu` is a header only library so `copts` will have no effect since there's no translation unit and nothing to actually compile.

I don't think the warnings set by `bootlin_toolchain` don't apply to `hip_cpu` since it uses `-isystem` (which it should imo):
https://bazel.build/reference/be/c-cpp#cc_binary.includes